common/include/common/utils/list-utils.h

```diff
@@ -34,10 +39,10 @@ class SinglyLinkedList
             return *this;
         }
 
-        self_type operator++([[maybe_unused]] int junk)
+        self_type operator++([[maybe_unused]] const int junk)
```

Comment:
why `const` here? You said `const` is only recommended for local variables declared in function body, not for function arguments, or did I misunderstood?

Reply:
Function arguments is considered pedantic. But I like it.